pyani_plus/workflows/snakemake_dnadiff.smk

```diff
@@ -61,32 +61,33 @@ rule filter:
     output:
         "{outdir}/{genomeA}_vs_{genomeB}.filter",
     shell:
-        "{params.delta_filter} -m {input} > {wildcards.outdir}/{wildcards.genomeA}_vs_{wildcards.genomeB}.filter"
+        "{params.delta_filter} -m {input} > {output}"
 
 
-# The filter rule runs show-diff wrapper for nucmer
+# The filter rule runs both show-diff and show-coords wrappers for nucmer
 # NOTE: This rule is used for dnadiff in the context of pyani_plus. The .filter file
-# is used to generate show-diff files, which are used to replicate AlignedBases.
-rule show_diff:
-    params:
-        show_diff=config["show_diff"],
-    input:
-        "{outdir}/{genomeA}_vs_{genomeB}.filter",
-    output:
-        "{outdir}/{genomeA}_vs_{genomeB}.qdiff",
-    shell:
-        "{params.show_diff} -qH {input} > {output}"
+# is used to generate both show-diff and show-coords files, which are used to
+# replicate AlignedBases, identity etc.
 
 
-# The filter rule runs show-coords wrapper for nucmer
-# NOTE: This rule is used for dnadiff in the context of pyani_plus. The .filter file
-# is used to generate show-coords files, which are used to replicate AlignedBases.
-rule show_coords:
+rule show_diff_and_coords:
     params:
+        db=config["db"],
+        show_diff=config["show_diff"],
         show_coords=config["show_coords"],
+        outdir=config["outdir"],
     input:
-        "{outdir}/{genomeA}_vs_{genomeB}.filter",
+        deltafilter="{outdir}/{genomeA}_vs_{genomeB}.filter",
+        genomeA=get_genomeA,
+        genomeB=get_genomeB,
     output:
-        "{outdir}/{genomeA}_vs_{genomeB}.mcoords",
+        qdiff="{outdir}/{genomeA}_vs_{genomeB}.qdiff",
+        mcoords="{outdir}/{genomeA}_vs_{genomeB}.mcoords",
     shell:
-        "{params.show_coords} -rclTH {input} > {output}"
+        """
+        {params.show_diff} -qH {input.deltafilter} > {output.qdiff}
+        {params.show_coords} -rclTH {input.deltafilter} > {output.mcoords} &&
+        .pyani-plus-private-cli log-dnadiff --database {params.db} \
```

Comment:
Will want to add chronic here (to the logging command only) as per #138, although given we control this command we can make it quiet by default.